scenario_player/node_support.py

```diff
@@ -510,9 +510,13 @@ def _monitor(runner: NodeRunner):
                     try:
                         runner.executor.check_subprocess()
                     except ProcessExitedWithError as ex:
+                        log.error("Node Error - Process", exception=ex)
```

Comment:
I'd use `log.exception()` that way the entire exception context will be logged.

Reply:
ah, my IDE didnt autocomplete on that, so I thought it wasnt available in `structlog`.